common.gypi

```diff
@@ -490,15 +490,6 @@
         'libraries': [ '-lelf' ],
       }],
       ['OS=="freebsd"', {
-        'conditions': [
-          ['"0" < llvm_version < "4.0"', {
-            # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial.
-            # Doesn't apply to llvm 4.0 (FreeBSD 11.1) or later.
```

Comment:
What about 11.0?

Reply:
I could put `>=11.2` (version 11.0 is EoL and version 11.1 will be EoL on September 30, 2018).

But we have this disclaimer in BUILDING.md: "The community does not build or test against end-of-life distributions (EoL)."

I think that by not being specific about the second part of the version number, we can more liberally upgrade it in CI as needed.

Reply:
Ah, makes sense, thanks for the clarification!